cqf-fhir-cr-hapi/src/test/java/org/opencds/cqf/fhir/cr/hapi/cdshooks/CdsServiceInterceptorTest.java

```diff
@@ -0,0 +1,111 @@
+package org.opencds.cqf.fhir.cr.hapi.cdshooks;
+
+import static ca.uhn.fhir.jpa.cache.ResourceChangeEvent.fromCreatedUpdatedDeletedResourceIds;
+import static java.util.Collections.emptyList;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+import static org.opencds.cqf.fhir.cr.hapi.cdshooks.CdsCrConstants.CDS_CR_MODULE_ID;
+
+import ca.uhn.fhir.jpa.cache.ResourceChangeEvent;
+import ca.uhn.hapi.fhir.cdshooks.api.json.CdsServiceJson;
+import ca.uhn.hapi.fhir.cdshooks.svc.CdsServiceRegistryImpl;
+import java.util.List;
+import org.hl7.fhir.instance.model.api.IIdType;
+import org.hl7.fhir.r4.model.IdType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InOrder;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opencds.cqf.fhir.cr.hapi.cdshooks.discovery.ICrDiscoveryService;
+import org.opencds.cqf.fhir.cr.hapi.cdshooks.discovery.ICrDiscoveryServiceFactory;
+
+@ExtendWith(MockitoExtension.class)
+class CdsServiceInterceptorTest {
+
+    @Mock
+    private CdsServiceRegistryImpl cdsServiceRegistry;
+
+    @Mock
+    private ICrDiscoveryServiceFactory discoveryServiceFactory;
+
+    @Mock
+    private ICdsCrServiceFactory crServiceFactory;
+
+    @InjectMocks
+    private CdsServiceInterceptor fixture;
```

Comment:
Suggestion:  call this "testSubject" for both tests instead

Reply:
done